README.md

```diff
@@ -1,69 +1,36 @@
-
-To complete the project, and make the test easier, a randomly-generated shape model is also
-available. In this dataset, some simple coloured geometric shapes are inserted into each picture,
-on a total random mode. There can be one rectangle, one circle and/or one triangle per image, or
-neither of them. Their location into each image is randomly generated (they just can't be too close
-to image borders). The shape and background colors are randomly generated as well.
-
-The picture below shows an example of image generated in this way:
+If you want to get more details on Oslandia activities around this topic, feel
+free to visit our [blog](http://oslandia.com/en/blog/). You certainly want to
+discover some of our results in the
+associated [web application](http://data.oslandia.io/deeposlandia):
 
-![Example of shape image](./images/shapes_example_2.png)
+![Web application homepage](./images/webapp_homepage.png)
 
 # Content
 
-The project contains some Python materials designed to illustrate the Tensorflow and Keras
-libraries (snippets and notebooks)
+The project contains following folders:
 
-+ [article](./article) contains the original text of articles that have been published
-  on [Oslandia blog](http://oslandia.com/en/blog/) on this topic
-+ [deeposlandia](./deeposlandia) contains main Python modules to train and test convolutional
-  neural networks
-+ [images](./images) contains some example images to illustrate the Mapillary dataset as well as
-  some preprocessing analysis results
-+ [notebooks](./notebooks) contains some Jupyter notebooks that aim at describing data or basic
-  neural network construction
-+ [tests](./tests) contains some test modules to guarantee the functioning of a bunch of snippets;
-  it uses the `pytest` framework.
++ [deeposlandia](./deeposlandia) contains main Python modules to train and test
+  convolutional neural networks
++ [examples](./examples) contains some Jupyter notebooks that aim at
+  describing data or basic neural network construction
++ [images](./images) contains some example images to illustrate the Mapillary
+  dataset as well as some preprocessing analysis results
++ [tests](./tests) contains some test modules to guarantee the functioning of a
+  bunch of snippets; it uses the `pytest` framework.
 
-Additionally, running the code may generate extra subdirectories in the data repository.
+Additionally, running the code may generate extra subdirectories in the chosen
+data repository.
 
 # Installation
 
 ## Requirements
 
-This project needs to load the following Python dependencies:
-
-+ cv2
-+ keras
-+ h5py
-+ logging
-+ matplotlib
-+ numpy
-+ pandas
-+ PIL
-+ tensorflow
-
-As a remark, the code has been run with Python3 (version 3.5). These dependencies are recalled in
-`setup.py` file, and additional dependencies for developing purpose are listed in
+The code has been run with Python3. The dependencies are recalled in `setup.py`
```

Comment:
Python 3 instead of Python3

Reply:
If prefer "All dependencies are specified in the `setup.py` file"...

Reply:
Modified!